google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/application/AppEngineAdminService.java

```diff
@@ -49,8 +73,54 @@ public static AppEngineAdminService getInstance() {
    * @throws IOException if there was a transient error connecting to the API
    */
   @Nullable
-  public abstract Application getApplicationForProjectId(@NotNull String projectId,
-      @NotNull Credential credential) throws IOException, GoogleApiException;
+  public Application getApplicationForProjectId(@NotNull String projectId,
```

Comment:
two thoughts: 
1) I thought we were only using an abstract class as a pseudo-interface that we could include a factory method on. (actually can we do that with an interface default method?)

2) It doesn't feel as though there are likely to be more than one subclasses of this abstract class.